### PR TITLE
Rename images to digital_objects

### PR DESCRIPTION
**Pull request by xhero**
I removed the (now) unused image tag and substituted it with digital_objects, could you have a look?

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -372,6 +372,7 @@ en:
     designer: Designer
     df_major: D♭ major
     df_minor: D♭ minor
+    digital_objects: Digital Objects
```
Comment (ahankinson) [inline locales/en.yml:375]: For English, the pattern is to have a capital first word, lower-case second word (See "Digitized music" below).

Comment (xhero) [reply locales/en.yml]: OK I update the file

**Comment by ahankinson** [comment]
I think the next thing we should do is try and figure out what to do here:

https://github.com/rism-ch/muscat/blob/develop/config/editor_profiles/default/configurations/SourceLabels.yml#L855-L856

At the first pass, it should probably be changed to:

```
doc_images:
    label: record.digital_objects
```

then the question is what to do about 'doc_images'...

**Comment by xhero** [comment]
I want to rename doc_images to doc_digital_objects in the guidelines: https://github.com/rism-ch/muscat-guidelines/blob/f039dd57cdeabced1c830cda5822f7970d703531/default/guidelines.yml#L215

And the the circle should be closed!

**Comment by xhero** [comment]
I updated the Muscat code:

```yml
doc_digital_objects:
  label: records.digital_objects
```
an update to the guidelines is pending

**Pull request merged**